helix-core/src/test/java/org/apache/helix/integration/rebalancer/WagedRebalancer/TestWagedRebalance.java

```diff
@@ -653,6 +674,36 @@ public void testRebalancerReset() throws Exception {
     Assert.assertFalse(newEV.equals(oldEV));
   }
 
+  @Test(dependsOnMethods = "test")
+  public void testRecreateSameResource() throws InterruptedException {
```

Comment:
This is not related to the code change, right?

Reply:
It is the issue that we want to fix.

Reply:
Without the change, this test will fail.